src/backend/joanie/settings.py

```diff
@@ -688,7 +688,10 @@ class Development(Base):
 
     ALLOWED_HOSTS = ["*"]
     CORS_ALLOW_ALL_ORIGINS = True
-    CSRF_TRUSTED_ORIGINS = ["http://localhost:3000", "http://localhost:8072"]
+    JOANIE_URL = values.Value(
+        "http://localhost:8072", environ_name="LOCALTUNNEL_URL", environ_prefix=None
+    )
+    CSRF_TRUSTED_ORIGINS = ["http://localhost:3000", JOANIE_URL]
```

Comment:
There is already a setting called `JOANIE_BACKOFFICE_BASE_URL`

```suggestion
    CSRF_TRUSTED_ORIGINS = ["http://localhost:3000", JOANIE_BACKOFFICE_BASE_URL]
```

Reply:
As seen together, replaced by 
```
    CSRF_TRUSTED_ORIGINS = [
        "http://localhost:3000",
        JOANIE_URL,
        Base.JOANIE_BACKOFFICE_BASE_URL,
    ]
```